roles/matrix-server/templates/riot-web/home.html.j2

```diff
@@ -0,0 +1,308 @@
+                </a>
+                <span class="mx_HomePage_comment">
+                    _t("Lots of rooms already exist in Matrix, linked to existing networks (Slack, IRC, Gitter etc) or independent. Check out the directory!")
+                 </span>
+            </div>
+        </div>
+{% if matrix_riot_web_welcome_user_id %}
+        <div class="mx_HomePage_row">
+            <div>
+                <h3>_t("Chat with Riot Bot")</h3>
+                <a class="mx_HomePage_room" href="#/user/{{ matrix_riot_web_welcome_user_id }}?action=chat">
+                    <img class="mx_HomePage_icon" src="home/rooms/riot-bot.png">
+                </a>
+                <span class="mx_HomePage_comment">
+                    _t("Get started with some tips from Riot Bot!")
+                </span>
+            </div>
+        </div>
+{% endif %}
+    </div>
+{% if matrix_riot_web_homepage_template_general %}
+
+    <h3>_t("General discussion about Matrix and Riot")</h3>
+
+    <div class="mx_HomePage_row">
+        <div class="mx_HomePage_room">
+            <a href="#/room/#matrix:matrix.org">
+                <img class="mx_HomePage_icon" src="home/rooms/matrix.png">
+                <span class="mx_HomePage_name">Matrix HQ</span>
+            </a>
+            <span class="mx_HomePage_desc">_t("Discussion of all things Matrix!")</span>
+        </div>
+        <div class="mx_HomePage_room">
+            <a href="#/room/#riot:matrix.org">
+                <img class="mx_HomePage_icon" src="home/rooms/riot.png">
+                <span class="mx_HomePage_name">Riot</span>
+            </a>
+            <span class="mx_HomePage_desc">_t("Riot/Web &amp; Desktop chat")</span>
+        </div>
+        <div class="mx_HomePage_room">
+            <a href="#/room/#riot-ios:matrix.org">
+                <img class="mx_HomePage_icon" src="home/rooms/riot-ios.png">
+                <span class="mx_HomePage_name">#riot-ios</span>
+            </a>
+            <span class="mx_HomePage_desc">_t("Riot/iOS &amp; matrix-ios-sdk chat")</span>
+        </div>
+        <div class="mx_HomePage_room">
+            <a href="#/room/#riot-android:matrix.org">
+                <img class="mx_HomePage_icon" src="home/rooms/riot-android.png">
+                <span class="mx_HomePage_name">#riot-android</span>
+            </a>
+            <span class="mx_HomePage_desc">_t("Riot/Android &amp; matrix-android-sdk chat")</span>
+        </div>
+    </div>
+{% endif %}
+{% if matrix_riot_web_homepage_template_technical %}
+
+    <h3>_t("Matrix technical discussions")</h3>
+    <h4>_t("Running Matrix services")</h4>
+
+    <div class="mx_HomePage_row">
+        <div class="mx_HomePage_room">
+            <a href="#/room/#matrix-docker-ansible-deploy:devture.com">
+                <img class="mx_HomePage_icon" src="home/rooms/matrix.png">
+                <span class="mx_HomePage_name">Synapse Support Community</span>
```

Comment:
This should be changed to "Matrix Docker Ansible Deploy".

Reply:
I missed that thanks.